source/deployment/cluster.rst

```diff
@@ -280,6 +280,31 @@ Transparent Failover
 
 The database can be configured for High Availability and transparent failover use the existing database technologies. We recommend MySQL Clustering, Postgres Clustering, or Amazon Aurora. Database transparent failover is beyond the scope of this documentation.
 
+Recommended Configuration Settings
+``````````````````````````````````
+
+If you're using Postgres as the choice of database, we recommend the following configuration optimizations on your Mattermost server.
+
+The following configuration was tested on an AWS Aurora r5.xlarge instance of Postgres 11.7.
+
+1. **max_connections**: If you are using read-replicas set reader connections to 1024, and writer connections to 256. If you are using a single instance, then only setting it to 1024 should be sufficient. If the instance of lower capacity than r5.xlarge, then set it to a lower number. Also tune the `MaxOpenConns` setting under the `SqlSettings` of Mattermost app accordingly.
+
+2. **random_page_cost**: Set it to 1.1, unless the DB is using spinning disks.
```

Comment:
Is this the same as saying, "do this if the DB is running on a solid state disk"?

Reply:
Lol yea :)